plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/PositionalXmlScanner.java

```diff
@@ -16,33 +16,30 @@
 
 package com.google.cloud.tools.eclipse.appengine.validation;
 
+import org.xml.sax.helpers.XMLReaderFactory;
+
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import javax.xml.parsers.ParserConfigurationException;
+import org.w3c.dom.Document;
 import org.xml.sax.InputSource;
 import org.xml.sax.SAXException;
 import org.xml.sax.XMLReader;
-import org.xml.sax.helpers.XMLReaderFactory;
 
-/**
- * Retrieves and returns the results of the SAX parser.
- */
-class PomParser {
+class PositionalXmlScanner {
   
-  static SaxParserResults readXml(byte[] bytes)
-      throws ParserConfigurationException, IOException, SAXException {
+  static Document parse(byte[] bytes) throws SAXException, ParserConfigurationException, IOException {
```

Comment:
I guess you no longer throw `ParserConfigurationException`. There are other places you re-throw or catch this, so you will want to fix them too.